sdk-core/transactions-outcome-parsers/token_management_transactions_outcome_parser.md

```diff
@@ -1 +1,115 @@
-// TBD: will be defined in a future PR.
+## TokenManagementTransactionsOutcomeParser
+
+A class that provides methods for extracting the output of token management transactions.
+i.e. Getting the ticker of a newly issued ESDT token.
+
+Each method should ensure that there is no `signalError` event;
+
+For methods that return multiple things, plain object can be used (JS) or simple DTOs(Py).
+
+For DTOs, a naming convention should be used. Something like name of the operation followed by the `Outcome` keywork.
+
+e.g.
+For the `parse_register_and_set_all_roles` method, we can use:
+```
+dto RegisterAndSetAllRolesOutcome:
+    token_identifier: str
+    roles: List[str]
+```
+
+```
+class TokenManagementTransactionsOutcomeParser:
+    // returns the token identifier
+    parse_issue_fungible(transaction_results_and_logs: ITransactionResultsAndLogsHolder): string;
```

Comment:
From the name of the class, we can say that the input argument can be `transaction_outcome: TransactionOutcome`.

Reply:
Renamed to `transaction_outcome: TransactionOutcome`